### ISSUE TITLE
Blenderbim - create project button freezes blender

### ISSUE THREAD

**Issue by Plae-Blue-Dot** [IfcOpenShell/IfcOpenShell#3720]
Using this version:
![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/78321676/d1bd4917-d1dd-4daa-95ce-bae5e31ac28a)

It just freezes
![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/78321676/11074bf9-8995-4289-93e8-5313614eeb22)

**Comment by Andrej730** [comment]
@Plae-Blue-Dot are you sure it's froze completely? It could be that it's just appending the entire IFC US Steel library to your new project. Could be related that order of templates changed a bit after I've renamed the AU Library to Furniture library in bc9b482 .

![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/9417531/eb628104-9083-4d97-96fe-4bdb3e7d6bb7)

**Comment by theoryshaw** [comment]
yes, the US steel template has a ton of types in it. I think it just takes forever to make the file.

**Comment by Moult** [comment]
Reopening until I move the libraries out of the starter templates and into a separate location.

**Comment by sboddy** [comment]
Looks like this can be closed as the libraries are no longer available in the starter templates.
